src/main/views/subscription-configure-list.njk

```diff
@@ -14,6 +14,8 @@
 {% block content %}
     <h1 class="govuk-heading-l">{{ title }}</h1>
```

Comment:
I'm getting an issue in the view where the filters return no results when you are in welsh. They work OK in english. Would you be able to have a look?

Reply:
`the issue should be fixed now i latest commit. Thanks form testing it :)